docs/source/howto/run_codes.rst

```diff
@@ -144,6 +144,38 @@ After the setup and configuration have been completed, let's check that everythi
 
 This command will perform various tests to make sure that AiiDA can connect to the computer, create new files in the scratch directory, retrieve files and query the job scheduler.
 
+.. _how-to:run-codes:computer:connection:
+
+Mitigating connection overloads
+----------------------------------
+
+Some compute resources, particularly large supercomputing centres, may not tolerate submitting too many jobs at once, executing scheduler commands too frequently or opening too many SSH connections.
+
+  * Limit the number of jobs in the queue.
+
+    Set yourself a limit for the maximum number of workflows to submit, and submit new ones only once previous workflows start to complete.
```

Comment:
> not actually possible to technically limit the total number of submitted workflows

not to the best of my knowledge, perhaps @sphuber can confirm

Reply:
Not actually sure what @csadorf means here. What do you mean with "just submit multiple times"? Anyway, what the text is trying to see is that AiiDA does not have an internal mechanism to limit the number of calculation jobs that are submitted to a computer's scheduler. So _if_ your machine's scheduler has a limit, then you should make sure that you do not submit a number of workflows that will submit a number of calculation jobs that exceeds that limit. Instead, just submit a maximum number of workflows to AiiDA that is guaranteed to never produce more than that limit in concurrent calcjobs

Reply:
What I mean is that I was previously under the assumption that AiiDA is capable to track (and thus limit) the maximum number of concurrently submitted jobs. Since this is not true, my questions with regarding to "cheating" that limitation by submitting multiple times in different sessions is void.